example/lib/main.dart

```diff
@@ -19,7 +19,7 @@ class MyApp extends StatelessWidget {
 }
 
 class MyHomePage extends StatefulWidget {
-  MyHomePage({Key key, this.title}) : super(key: key);
+  MyHomePage({Key? key, required this.title}) : super(key: key);
```

Comment:
Did you mean to `@required` attribute?

Reply:
In null safety Dart 2.12/Flutter 2 the `required` is a keyword not an annotation.